README.md

```diff
@@ -129,8 +129,48 @@ New tenants will come with some basic initial data already installed, but if you
 3. use Ctrl + D or `exit()` to close the Python shell.
 
 
-## Contributing
+### Enabling Google Sign In (Optional)
 
-See [CONTRIBUTING.md](https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md) if you plan to contribute code to this project.  It contains critical information for your pull request to be accepted and will save you a lot of time!
 
+Here are the steps, assuming that you now have a functional tenant:
+
+1. Obtain Google credentials: https://developers.google.com/workspace/guides/create-credentials#oauth-client-id
+2. Make sure that in the Authorized URIs, add `http://hackerspace.localhost.net:8000/accounts/google/login/callback/`. We will explain why we are using `localhost.net` later but for now, just add this.
+3. Go to Social Applications: http://localhost:8000/admin/socialaccount/socialapp/
+4. Click Add Social Application
+5. Fill in `Client Id` and `Secret Key`. And then add the `Available Sites` to `Chosen Sites`
+6. Click Save
+7. Go to your Site Configuration: http://hackerspace.localhost:8000/config/ and click `Enable sign-in via Google`
+8. Done
+
+When you are developing locally, Google won't allow you to add `http://hackerspace.localhost:8000/accounts/google/login/callback/` in the Authorized URIs. So we need a way to bypass this in our local machine by mapping
+our localhost to `localhost.net` so we can access our tenant via `http://hackerspace.localhost.net:8000`.
+
+We need to modify our hosts file aka `/etc/hosts`. You can also take a look at this [tutorial](https://www.howtogeek.com/27350/beginner-geek-how-to-edit-your-hosts-file/).
+
+1. Add the following, preferably at the bottom of the file:
+
+```conf
+127.0.0.1 localhost.net hackerspace.localhost.net
+```
+
+2. We need to update the`ALLOWED_HOSTS` in our .env file:
 
+```bash
+ALLOWED_HOSTS=.localhost,.localhost.net
+```
+
+3. For the final step, we need to let `django-tenants` know that `hackerspace.localhost.net` is also a valid domain.
+Run `$ ./src/manage.py shell` and type in the following commands
```

Comment:
I'm confused about this step. Do I need to access the shell in production every time a deck owner wants Google Sign in? I thought this was done automatically... or we could do from public tenant admin or something? I remember us discussing this but now I'm confused again =)

Reply:
This is just for dev environment isn't it?

Reply:
Ahh. This must've happened when I was fixing a merge conflict. Let me fix this first